app/presenters/tree_node_builder.rb

```diff
@@ -474,10 +474,9 @@ def miq_report_node(text, image, name, status)
   def assigned_server_role_node(object)
     @node = {
       :key      => build_object_id,
-      :addClass => "dynatree-title",
       :title    => options[:tree] == :servers_by_role_tree ?
-        "#{Dictionary.gettext('MiqServer', :type => :model, :notfound => :titleize)}: #{object.name} [#{object.id}]" :
-        "Role: #{object.server_role.description}"
+        "<strong>#{Dictionary.gettext('MiqServer', :type => :model, :notfound => :titleize)}: #{object.name} [#{object.id}]</strong>" :
```

Comment:
@mzazrivec : this can be replaced with a gettext call on a string constant, rigtht?